purchasing/templates/opportunities/front/expired.html

```diff
@@ -0,0 +1,49 @@
+{% extends "opportunities/layout.html" %}
+
+{% block page_title %}
+   {{ super() }} - Expired City of Pittsburgh opportunities
+{% endblock %}
+
+{% block content %}
+<div class="container">
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2">
+
+      {% include 'includes/flashes.html' %}
+
+      <form class="form form-horizontal" method="POST" action="#">
```

Comment:
Absolutely nothing! Cribbed this from the available opportunities table and missed this form block.